content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -575,7 +577,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
   completions without relying on Pods remaining in the cluster indefinitely.
   The Job controller uses Pod finalizers and a field in the Job status to keep
   track of the finished Pods to count towards completion.
+- `KMSv1`: Enables KMS v1 API for encryption at rest. See [Using a KMS Provider for data encryption](/docs/tasks/administer-cluster/kms-provider) for more details.
```

Comment:
```suggestion
- `KMSv1`: Enables KMS v1 API (deprecated) for encryption at rest. See [Using a KMS Provider for data encryption](/docs/tasks/administer-cluster/kms-provider) for more details.
```

(nit) I would direct people to read https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/ (and then find the task page if they need to)